build/Makefile

```diff
@@ -2,6 +2,9 @@
 POSSIBLE_TARGETS := module unit_tests run_unit_tests wheel sdist
 export POSSIBLE_TARGETS
 
+ADDITIONAL_POSSIBLE_TARGETS := test flake8
```

Comment:
Now we have POSSIBLE_TARGETS and ADDITIONAL_POSSIBLE_TARGETS
What is the distinction between them? Seems arbitrary, why even have two variables?